packages/sonarwhal/src/lib/cli/analyze.ts

```diff
@@ -168,8 +169,12 @@ export default async (actions: CLIOptions): Promise<boolean> => {
         }
     }
 
-    const config = SonarwhalConfig.fromConfig(userConfig, actions);
-    const resources = resourceLoader.loadResources(config);
+    const config: SonarwhalConfig = SonarwhalConfig.fromConfig(userConfig, actions);
+    const resources: SonarwhalResources = resourceLoader.loadResources(config);
+
+    // Question: Should we track this after all packages has been installed?
+    trackEvent('analyze', config);
+    // Question: Track missing resources?
```

Comment:
We should track at least: 

* [ ] If sonarwhal is running globally or not
* [ ] Missing or invalid resources
* [ ] Configuration used

Reply:
@sonarwhal/contributors any idea how to know if sonarwhal is running globally or not?